File: support/src/main/java/com/wuguangxin/mvp/contract/FileModel.java
package com.wuguangxin.mvp.contract;

/**
 * 文件M实现
 * Created by wuguangxin on 2020-03-19.
 */
public class FileModel implements FileContract.Model {

    @Override
    public void uploadFile(String filePath, FileContract.Callback callback) {
        // http request..
    }

    @Override
    public void deleteFile(String fileUrl, FileContract.Callback callback) {
        // http request..
    }
}

File: support/src/main/java/com/wuguangxin/mvp/contract/FileModel.kt
package com.wuguangxin.mvp.contract

/**
 * 文件M实现
 * Created by wuguangxin on 2020-03-19.
 */
class FileModel : FileContract.Model {
    override fun uploadFile(filePath: String?, callback: FileContract.Callback?) {
        // request..
    }

    override fun deleteFile(url: String?, callback: FileContract.Callback?) {
        // request..
    }
}